docs/source/get_started/new_project.md

```diff
@@ -112,6 +112,11 @@ You can also enter this in a single line as follows:
 kedro new --name=testproject --tools=lint,docs,pyspark --example=n
 ```
 
+### Telemetry consent
+
+The `--telemetry` flag offers the option to register consent to have user analytics collected in the moment of the creation of the project. This option bypasses the prompt to collect analytics that would otherwise appear on the moment the `kedro` command is invoked for the first time inside the project. In case the `--telemetry` flag is not used, the user will be prompted to accept or reject analytics collection as usual.
+
+When creating your new Kedro project, use the values `yes` or `no` to register consent to have user analytics collected for this specific project.
```

Comment:
I believe here and for example_pipeline, we should leave it as it is: with yes/no option. Because if you want to choose `--telemetry=yes`,` --skip-telemetry` will not provide you that option.

Reply:
I'm happy with `--telemetry yes/no` 👍🏽